Source/Frontend/UI/Forms/ConnectionStatusForm.cs

```diff
@@ -6,23 +6,23 @@
     using RTCV.Common;
     using RTCV.UI.Modular;
 
-    public partial class RTC_ConnectionStatus_Form : ComponentForm, IAutoColorize, IBlockable
+    public partial class ConnectionStatusForm : ComponentForm, IAutoColorize, IBlockable
     {
         public new void HandleMouseDown(object s, MouseEventArgs e) => base.HandleMouseDown(s, e);
         public new void HandleFormClosing(object s, FormClosingEventArgs e) => base.HandleFormClosing(s, e);
 
-        public RTC_ConnectionStatus_Form()
+        public ConnectionStatusForm()
         {
             InitializeComponent();
-            this.Shown += RTC_ConnectionStatus_Form_Shown;
-            this.btnTriggerKillswitch.MouseClick += BtnTriggerKillswitch_MouseClick;
+            this.Shown += OnFormShown;
+            this.btnTriggerKillswitch.MouseClick += OnTriggerKillswitchMouseClick;
         }
 
         private readonly string[] _flavorText = {
             "Imagine if we had actual flavor text",
             "Fun flavor text goes here",
             "The V probably stands for Vanguard",
-            "What are naming conventions",
+            "Now with naming conventions!",
```

Comment:
I thought this change was appropriate given the work I've been doing 😃 

I'll revert if preferred

Reply:
LGTM